announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementAction.java

```diff
@@ -2116,11 +2135,11 @@ protected String buildShowMetadataContext(VelocityPortlet portlet, Context conte
 		}
 		catch (IdUnusedException e)
 		{
-			if (log.isDebugEnabled()) log.debug("{}.buildShowMetadataContext()", this, e);
+			log.debug("{}.buildShowMetadataContext()", this, e);
 		}
 		catch (PermissionException e)
 		{
-			if (log.isDebugEnabled()) log.debug("{}.buildShowMetadataContext()", this, e);
+			log.debug("{}.buildShowMetadataContext()", this, e);
```

Comment:
no need to use class or method name in log as it is already configured to do that